transforms/language/doc_quality/ray/Makefile

```diff
@@ -0,0 +1,57 @@
+# Define the root of the local git clone for the common rules to be able 
+# know where they are running from.
+REPOROOT=../../../..
+# Include a library of common .transform.* targets which most
+# transforms should be able to reuse.  However, feel free
+# to override/redefine the rules below. 
+include $(REPOROOT)/transforms/.make.transforms
+
+TRANSFORM_NAME=doc_quality
+# $(REPOROOT)/.make.versions file contains the versions
+
+BASE_IMAGE=${RAY_BASE_IMAGE}
```

Comment:
Not sure about whether we should keep this.  On one hand it allows the base image to be changed from the .make.defaults, but it is currently not consistent with other transforms.  So, for now can you remove BASE_IMAGE.

Reply:
Removed.

Reply:
@daw3rd Unfortunately, current build system seems to have a problem in removing `BASE_IMAGE=${RAY_BASE_IMAGE}`

```
 make DOCKER_IMAGE=doc_quality-ray:latest .defaults.image
make[5]: Entering directory '/home/runner/work/data-prep-kit/data-prep-kit/transforms/language/doc_quality/ray'
# The following touch seems to be needed to work around a docker build problem in which
# it seemed to be using a pyproject from a previously built image.  Specifically, malware's
# ray pyproject.toml seemed to be copied into the image instead of the proglang_select python version.
# This could not be reproduced locally and was only seen when running as the build-image.yml workflow
#if [ -e pyproject.toml ]; then \
#	cat pyproject.toml;	\
#fi
if [ -e pyproject.toml ]; then 	\
	touch pyproject.toml;	\
fi
docker build -t doc_quality-ray:latest		\
	--build-arg EXTRA_INDEX_URL=	\
	--build-arg BASE_IMAGE=  		\
	--build-arg BUILD_DATE=2024-07-25T00:14:00Z  	\
	--build-arg GIT_COMMIT=f7ee784 . 
#0 building with "default" instance using docker driver

#1 [internal] load build definition from Dockerfile
#1 transferring dockerfile: 1.30kB done
#1 DONE 0.0s
Dockerfile:3
--------------------
   1 |     ARG BASE_IMAGE=docker.io/rayproject/ray:2.24.0-py310
   2 |     
   3 | >>> FROM ${BASE_IMAGE}
   4 |     
   5 |     # install pytest
--------------------
```

Reply:
sorry, my bad. i was wrong on this.  othre transforms do use this.